packages/ui/src/modal/Progress.tsx

```diff
@@ -36,12 +37,13 @@ const Progress: FC<Props> = ({ title, txHashes, blockExplorerBaseUrl }) => {
       </Box>
       {hasTxHashes ? (
         <Flex direction="column" align="center" css={{ gap: '$2' }}>
-          {txHashes?.map((txHash, index) => {
-            const truncatedTxHash = truncateAddress(txHash)
+          {txHashes?.map((hash, index) => {
+            const truncatedTxHash = truncateAddress(hash.txHash)
+            const blockExplorerBaseUrl = getChainBlockExplorerUrl(hash.chainId)
```

Comment:
should we call this function in a memo or is it lightweight enough that it's ok to call it on every render?